apps/redi-connect/src/components/organisms/EditableMentoringTopics.tsx

```diff
@@ -44,56 +44,118 @@ const categoriesByGroup = groupBy(CATEGORIES, (category) => category.group)
 
 const formCategoryGroups = objectEntries(CATEGORY_GROUPS)
 
+const getInitialValues = ({
+  userType,
+  categories,
+}: {
+  userType: ProfileUserType
+  categories: MentoringTopic[]
+}) => {
+  const groups = Object.keys(
+    CATEGORY_GROUPS
+  ) as unknown as (keyof typeof CATEGORY_GROUPS)[]
+
+  const emptyGroups = Object.fromEntries(
+    groups.map((key) => [key, []])
+  ) as Record<keyof typeof CATEGORY_GROUPS, []>
```

Comment:
We seem to be using `keyof typeof CATEGORY_GROUPS` at least two times. I think it's worth to create a type for this where CATEGORY_GROUPS is imported from, and use that here instead.